ci/build.yml

```diff
@@ -6,14 +6,14 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tags: 1.10.0
+    tag: 1.12.12
```

Comment:
We can upgrade to 1.13.7

Also does it matter that it is `tag` instead of `tags`?

Reply:
I didn't see that I changed tags to tag, thanks!